Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -17432,7 +17432,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "echo \"Preprocessing sandbox\"\nScripts/generate-derived-sources.sh sandbox-profiles-ios\nmkdir -p ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.adattributiond.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.webpushd.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.GPU.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.Networking.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.WebAuthn.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.WebContent.sb ${DSTROOT}/${INSTALL_PATH}\n";
+			shellScript = "echo \"Preprocessing sandbox\"\nScripts/generate-derived-sources.sh sandbox-profiles-ios\nmkdir -p ${DSTROOT}/${INSTALL_PATH}\n\nSANDBOX_GPU=${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.GPU.sb\nSANDBOX_NETWORKING=${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.Networking.sb\nSANDBOX_WEBCONTENT=${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.WebContent.sb\n\nsbutil compile ${SANDBOX_GPU}\nsbutil compile ${SANDBOX_NETWORKING}\nsbutil compile ${SANDBOX_WEBCONTENT}\n\ncp ${SANDBOX_GPU} ${DSTROOT}/${INSTALL_PATH}\ncp ${SANDBOX_NETWORKING} ${DSTROOT}/${INSTALL_PATH}\ncp ${SANDBOX_WEBCONTENT} ${DSTROOT}/${INSTALL_PATH}\n\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.adattributiond.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.webpushd.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.WebAuthn.sb ${DSTROOT}/${INSTALL_PATH}\n";
```

Comment:
For anyone else looking at this, here's the shell script unescaped:
```
echo "Preprocessing sandbox"
Scripts/generate-derived-sources.sh sandbox-profiles-ios
mkdir -p ${DSTROOT}/${INSTALL_PATH}

SANDBOX_GPU=${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.GPU.sb
SANDBOX_NETWORKING=${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.Networking.sb
SANDBOX_WEBCONTENT=${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.WebContent.sb

sbutil compile ${SANDBOX_GPU}
sbutil compile ${SANDBOX_NETWORKING}
sbutil compile ${SANDBOX_WEBCONTENT}

cp ${SANDBOX_GPU} ${DSTROOT}/${INSTALL_PATH}
cp ${SANDBOX_NETWORKING} ${DSTROOT}/${INSTALL_PATH}
cp ${SANDBOX_WEBCONTENT} ${DSTROOT}/${INSTALL_PATH}

cp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.adattributiond.sb ${DSTROOT}/${INSTALL_PATH}
cp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.webpushd.sb ${DSTROOT}/${INSTALL_PATH}
cp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.WebAuthn.sb ${DSTROOT}/${INSTALL_PATH}
```

Reply:
Those are good point, fixed in latest patch!

Thanks for reviewing!

Reply:
I agree with both of Elliott's suggestions!